build.gradle

```diff
@@ -56,11 +56,9 @@ checkstyle {
 }
 
 pmd {
-  toolVersion = "6.55.0"
+  toolVersion = "7.0.0"
```

Comment:
Something really minor. Other projects upgraded checkstyle version to 10.15.0. If u rather this to be done in the next dependency upgrade I dont mind too.